inventory/main/src/main/java/org/opennms/horizon/inventory/service/taskset/ScannerTaskSetService.java

```diff
@@ -136,6 +137,21 @@ Optional<TaskDefinition> createDiscoveryTask(List<String> ipAddresses, Long loca
                 } catch (Exception e) {
                     LOG.error("Not able to parse IP range from {}", ipAddressDTO);
                 }
+            } else if (ipAddressDTO.contains("/")) {
+                try {
+                    var utils = new SubnetUtils(ipAddressDTO);
+                    utils.setInclusiveHostCount(true);
+                    var info = utils.getInfo();
+                    var lowAddress = info.getLowAddress();
+                    var highAddress = info.getHighAddress();
+                    var ipRangeBuilder = IpRange.newBuilder();
+                    ipRangeBuilder.setBegin(lowAddress);
+                    ipRangeBuilder.setEnd(highAddress);
```

Comment:
Nit: This could be more succinctly specified:

```suggestion
                    var ipRangeBuilder = IpRange.newBuilder()
                        .setBegin(info.getLowAddress())
                        .setEnd(info.getHighAddress());
```